wormhole-connect/src/hooks/useFetchTransactionHistory.ts

```diff
@@ -0,0 +1,310 @@
+import { useCallback, useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { amount as sdkAmount, chainIdToChain } from '@wormhole-foundation/sdk';
+
+import config from 'config';
+import { getTokenById, getTokenDecimals, getWrappedToken } from 'utils';
+
+import type { RootState } from 'store';
+import type { Chain } from '@wormhole-foundation/sdk';
+import type { RelayerFee } from 'store/relay';
+
+export interface Transaction {
+  // Transaction hash
+  txHash: string;
+
+  // Stringified addresses
+  sender?: string;
+  recipient: string;
+
+  amount: string;
+  amountUsd: number;
+
+  toChain: Chain;
+  fromChain: Chain;
+
+  // Source token address
+  tokenAddress: string;
+  tokenKey: string;
+  tokenDecimals?: number;
+
+  // Destination token
+  receivedTokenKey?: string;
+  receiveAmount?: string;
+  relayerFee?: RelayerFee;
+
+  // Amount of native gas being received, in destination gas token units
+  // For example 1.0 is 1.0 ETH, not 1 wei
+  receiveNativeAmount?: number;
+
+  // Timestamps
+  senderTimestamp?: string;
+  receiverTimestamp?: string;
+}
+
+interface WormholeScanTransaction {
```

Comment:
I don't think we need to export as we don't need to use it anywhere else (unless we decide to move out the parsing logic to somewhere else in the future).